homework/assignment_5/baseball_features.sql

```diff
@@ -0,0 +1,206 @@
+#####################################################################
+# Select database
+#####################################################################
+USE baseball;
+SHOW tables FROM baseball;
+show columns from pitcher_stat;
+show columns from pitcher_counts;
+show columns from pitchersInGame;
+
+#####################################################################
+# HOME TEAM WINS
+#####################################################################
+CREATE OR REPLACE TABLE home_team_wins
```

Comment:
creating tables takes pretty long, each one a few minutes. Could be quicker just using select statement